### PR TITLE
doc: update streaming docs

### PR DESCRIPTION
**Pull request by sam-github**
WIP

### CODE REVIEW

File: Streaming/Streaming-To-Youtube.md [hunk 1]
```diff
@@ -3,20 +3,29 @@
...
 ## Prerequisites
-You need the Foundation login credentials for Zoom in order to host the meeting. In addition, your YouTube Account must be a manager of the [Node.js YouTube account](https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ). Ask the TSC Chair for access to Zoom and YouTube.
+
+You need the Foundation login credentials for Zoom in order to host the meeting.
+The Zoom user ID is `nodejs-meetings@linuxfoundation.org`, the password needs
+to be communicated privately, ask the TSC Chair.
+
+Your YouTube Account must be a manager of the
+[Node.js YouTube account](https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ). 
+Ask the TSC Chair to make your account a manager.
```
Comment (sam-github) [inline Streaming/Streaming-To-Youtube.md]: How do I verify that my account is an account manager? I've looked all over through my youtube settings, and see no sign that I'm an admin.

File: Streaming/Streaming-To-Youtube.md [hunk 2]
```diff
@@ -3,20 +3,29 @@
 We publicly live stream our meetings to YouTube using [Zoom](https://zoom.us/).
 
 ## Prerequisites
-You need the Foundation login credentials for Zoom in order to host the meeting. In addition, your YouTube Account must be a manager of the [Node.js YouTube account](https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ). Ask the TSC Chair for access to Zoom and YouTube.
+
+You need the Foundation login credentials for Zoom in order to host the meeting.
```
Comment (sam-github) [inline Streaming/Streaming-To-Youtube.md]: It says "you need them", but its not clear - when hosting a meeting, do we need to ensure we logout from zoom.us with our normal credentials, and log back in with the foundation ones?

Comment (mhdawson) [reply Streaming/Streaming-To-Youtube.md]: Yes you must be logged in with the foundation credential to be able to start the meeting.

File: Streaming/Streaming-To-Youtube.md (hunk 1 above)
Comment (mhdawson) [reply Streaming/Streaming-To-Youtube.md]: There are more people who can do this it includes:
Amanda Ennis (foundation),
Brian Warner (foundation),
Rachel Romoff (foundation)
James Snell
Jeremiah Senkpiel
Rod Vagg 
Tracy Hinds 

The directions should probably be to ask Amanda, Brian or Rachel as our foundation managers of the account. 

In order to get to the right page its:

youtube.com
click on the node.js icon on the right top, select settings
select "Add or remove managers"
select "Manage permissions"
On that  page you can use the +people at the top right of the popup to add people.  It also list all those in the list not sure what you can see if you are a manager versus and owner.

If you are not a manage I believe you won't even have the node.js icon that you can select.

**Comment by sam-github** [comment]
Updated with info from comments, can I merge?

**Pull request merged**